lib/html-proofer/check/links.rb

```diff
@@ -79,15 +79,15 @@ def check_schemes(link, line, content)
   end
 
   def handle_mailto(link, line, content)
-    if link.path.empty?
+    if link.path.nil?
```

Comment:
For this and L90, I'd prefer the revert. I believe In the even of the path being missing, the `path` attribute is kept as `""`, which, while not `nil`, is an empty string, and that is wrong in my opinion.